SparkPost/2web2ui
# SA-1596: Disabled timezone and precision if not on metrics rollup

### What Changed
 - Disabled timezone and precision if metrics rollup feature flag not enable

### How To Test
 - Enable hibana on an account
 - Verify that timezone and precision are not modifiable
 - Enable user feature flag `use-metrics-rollup`
 - Timezone and precision should now change

### To Do
